charmcraft/commands/init.py

```diff
@@ -37,6 +37,7 @@
 .
 ├── metadata.yaml        - Charm operator package description
 ├── README.md            - Frontpage for your charmhub.io/charm/
```

Comment:
We should specify it should be written from the PoV of the charm user

Reply:
I don't think I'm following you here. `charmcraft init` will only be used by Charm authors.